charts/k8s-monitoring/default_allow_lists/README.md

```diff
@@ -0,0 +1,27 @@
+# Default Allow Lists
```

Comment:
Also maybe worth:

- adding a sentence explaining that any additional custom scraping that are added could limit metrics based on usage of `action = "keep"`
- adding a link over to custom metrics tuning